test/e2e/provisioning/helper.go

```diff
@@ -125,6 +126,20 @@ func testScenario(t *testing.T, testCase scenario, cloudProvider string, testPar
 	scenarioParams = append(scenarioParams, fmt.Sprintf("<< KUBERNETES_VERSION >>=%s", testCase.kubernetesVersion))
 	scenarioParams = append(scenarioParams, fmt.Sprintf("<< YOUR_PUBLIC_KEY >>=%s", os.Getenv("E2E_SSH_PUBKEY")))
 
+	rhelSubscriptionManagerUser := os.Getenv("RHEL_SUBSCRIPTION_MANAGER_USER")
+	rhelSubscriptionManagerPassword := os.Getenv("RHEL_SUBSCRIPTION_MANAGER_PASSWORD")
+	rhsmOfflineToken := os.Getenv("REDHAT_SUBSCRIPTIONS_OFFLINE_TOKEN")
+
+	if testCase.osName == string(providerconfigtypes.OperatingSystemRHEL) {
+		if rhelSubscriptionManagerUser == "" || rhelSubscriptionManagerPassword == "" || rhsmOfflineToken == "" {
+			t.Errorf("Unable to run kubevirt tests, KUBEVIRT_E2E_TESTS_KUBECONFIG, RHEL_SUBSCRIPTION_MANAGER_USER, " +
```

Comment:
In all other error messages about missing env vars, we only mention the relevant ones for the specific check. I would therefore not mention `KUBEVIRT_E2E_TESTS_KUBECONFIG` here again, its existence has already been checked. Maybe phrase it `Unable to run kubevirt tests with RHEL, .... must be set.`.

Reply:
Yeah, that's a copy and paste mistake ;). Kubevirt shouldn't be mentioned here because this message now is generic for all cloud providers thus should be something like: 
`Unable to run e2e tests, RHEL_SUBSCRIPTION_MANAGER_USER, RHEL_SUBSCRIPTION_MANAGER_PASSWORD, and REDHAT_SUBSCRIPTIONS_OFFLINE_TOKEN must be set when rhel is used as an os`.